app/src/split/bluetooth/service.c

```diff
@@ -45,12 +48,68 @@ BT_GATT_SERVICE_DEFINE(
     BT_GATT_DESCRIPTOR(BT_UUID_NUM_OF_DIGITALS, BT_GATT_PERM_READ, split_svc_num_of_positions, NULL,
                        &num_of_positions), );
 
+#define SERVICE_STACK_SIZE 512
+#define SERVICE_PRIORITY 5
```

Comment:
Should these be defined inline here or at the top or in `service.h`?

Should they be Kconfig variables?

Reply:
Moved to Kconfig.

Reply:
Indeed. Moved to Kconfig.